pytorch/inference/docker/1.9/py3/Dockerfile.graviton

```diff
@@ -1,4 +1,139 @@
+ && rm ~/miniconda.sh
+
+RUN /opt/conda/bin/conda update conda
+
+RUN /opt/conda/bin/conda install -c conda-forge python=$PYTHON_VERSION
+
+RUN /opt/conda/bin/conda install -y \
+    ruamel_yaml \
+    cython \
+    ipython \
+    numpy \
+    parso \
+    scipy
+
+RUN /opt/conda/bin/conda clean -ya
+
+RUN wget https://www.open-mpi.org/software/ompi/v4.0/downloads/openmpi-$OPEN_MPI_VERSION.tar.gz \
+ && gunzip -c openmpi-$OPEN_MPI_VERSION.tar.gz | tar xf - \
+ && cd openmpi-$OPEN_MPI_VERSION \
+ && ./configure --prefix=/home/.openmpi \
+ && make all install \
+ && cd .. \
+ && rm openmpi-$OPEN_MPI_VERSION.tar.gz \
+ && rm -rf openmpi-$OPEN_MPI_VERSION
+
+# The ENV variables declared below are changed in the previous section
+# Grouping these ENV variables in the first section causes
+# ompi_info to fail. This is only observed in CPU containers
+ENV PATH="$PATH:/home/.openmpi/bin"
+ENV LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/home/.openmpi/lib/"
+RUN ompi_info --parsable --all | grep mpi_built_with_cuda_support:value
+
+RUN conda install -c \
+    conda-forge \
+    opencv \
+ && conda install -y \
+    scikit-learn \
+    pandas \
+    requests \
+ && conda clean -ya \
+ && pip install --upgrade pip --trusted-host pypi.org --trusted-host files.pythonhosted.org \
+ && ln -s /opt/conda/bin/pip /usr/local/bin/pip3 \
+ && pip install packaging==20.4 \
+    enum-compat==0.0.3 \
+    "cryptography>=3.3.2"
+
+# install torch and torchvision from the PyTorch website
+RUN pip install torch torchvision
+
+RUN conda install -y -c conda-forge "pyyaml>=5.4,<5.5"
+RUN pip install pillow==8.2.0 "awscli<2"
+RUN pip install --no-cache-dir "sagemaker-pytorch-inference>=2"
+
+RUN pip uninstall -y model-archiver multi-model-server \
+ && pip install captum \
+ && pip install torchserve==$TS_VERSION \
+ && pip install torch-model-archiver==$TS_VERSION
+
+RUN cd tmp/ \
+ && rm -rf tmp*
+
+RUN useradd -m model-server \
+ && mkdir -p /home/model-server/tmp /opt/ml/model \
+ && chown -R model-server /home/model-server /opt/ml/model
+
+COPY torchserve-entrypoint.py /usr/local/bin/dockerd-entrypoint.py
+COPY config.properties /home/model-server
+
+RUN chmod +x /usr/local/bin/dockerd-entrypoint.py
+
+COPY deep_learning_container.py /usr/local/bin/deep_learning_container.py
+
+RUN chmod +x /usr/local/bin/deep_learning_container.py
+
+RUN HOME_DIR=/root \
+ && curl -o ${HOME_DIR}/oss_compliance.zip https://aws-dlinfra-utilities.s3.amazonaws.com/oss_compliance.zip \
+ && unzip ${HOME_DIR}/oss_compliance.zip -d ${HOME_DIR}/ \
+ && cp ${HOME_DIR}/oss_compliance/test/testOSSCompliance /usr/local/bin/testOSSCompliance \
+ && chmod +x /usr/local/bin/testOSSCompliance \
+ && chmod +x ${HOME_DIR}/oss_compliance/generate_oss_compliance.sh \
+ && ${HOME_DIR}/oss_compliance/generate_oss_compliance.sh ${HOME_DIR} ${PYTHON} \
+ && rm -rf ${HOME_DIR}/oss_compliance*
+
+RUN curl https://aws-dlc-licenses.s3.amazonaws.com/pytorch-1.8/license.txt -o /license.txt
```

Comment:
Please change this to https://aws-dlc-licenses.s3.amazonaws.com/pytorch-1.9/license.txt